README.md

```diff
@@ -40,131 +40,152 @@ FIEDLER, Niklas, et al. [ImageTagger: An Open Source Online Platform for Collabo
    organization={Springer}
 }
 ```
-## Upgrade
 
-```
-pip install -U -r requirements.txt
-./manage.py migrate
-```
-
-for additional steps on some releases see instructions
-in [UPGRADE.md](https://github.com/bit-bots/imagetagger/blob/master/UPGRADE.md)
-
-## Install
-
-Checkout the latest release:
-
-```
-git checkout v0.x
-```
-
-In our production Senty is used for error reporting (pip install sentry-sdk).
-django-auth-ldap is used for login via ldap
-uwsgi is used to serve the app to nginx
-
-Install Python Dependencies:
-
-```
-pip3 install -r requirements.txt
-```
-
-Copy settings.py.example to settings.py in the imagetagger folder:
-
-```
-cp imagetagger/settings.py.example imagetagger/settings.py
-```
-
-and customize the settings.py.
-
-The following settings should probably be changed:
-
-+ The secret key
-+ The DEBUG setting
-+ The ALLOWED\_HOSTS
-+ The database settings
-+ The UPLOAD\_FS\_GROUP to the id of the group that should access and create the uploaded images
-
-For the database, postgresql is used. Install it by running `sudo apt install postgresql` on Debian based operating systems. A default database cluster is automatically initialized.
-
-Other systems may require different commands to install the package and the database cluster may
-have to be initialized manually (e.g. using `sudo -iu postgres initdb --locale en_US.UTF-8 -D '/var/lib/postgres/data'`).
-
-To start the postgresql server, run `sudo systemctl start postgresql.service`. If the server should always be started on boot, run `sudo systemctl enable postgresql.service`.
-
-Then, create the user and the database by running
-
-`sudo -iu postgres psql`
-
-and then, in the postgres environment
-
-```
-CREATE USER imagetagger PASSWORD 'imagetagger';
-CREATE DATABASE imagetagger WITH OWNER imagetagger ENCODING UTF8;
-```
-
-where of course the password and the user should be adapted to the ones specified in the database settings in the settings.py.
-
-To initialize the database, run `./manage.py migrate`
-
-To create an administrator user, run `./manage.py createsuperuser`.
-
-`./manage.py runserver` starts the server with the configuration given in the settings.py file.
-
-To create annotation types, log into the application and click on Administration at the very bottom of the home page.
```

Comment:
This line should be kept.

Reply:
I don't really see a good position in the document.

I would not put it right there because it applies to every deployment type not only when running locally.

Reply:
Maybe add it at `Configuration`? I guess you could say that it is still a part of the configuration, it just is not done in code.